platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -99,10 +99,20 @@
 	 */
 	public static native void hover(int type, int x, int y);
 
+	/**
+	 * Forward mouse pressed events from the main thread to the GL thread.
+	 */
+	public static native void mouse_pressed(int button, int x, int y, boolean pressed);
```

Comment:
me at 3am thinking about a name for a function... thinking about changing it to `mouse_click` and `mouse_drag` or somthing on that line

Reply:
but mouse click make no sense if it also handle mouse release

Reply:
Well `mouseEvent` seems to encapsulate all since a `click` is a combination of mouse press and release events and a `drag` is a combination of mouse press, move and release events.

Reply:
currently `drag` is not the right name, it's more of a move while pressed, which happans kinda independently... I think it's a shame to combine them and it will make readibilty wise less sense